package.json

```diff
@@ -66,6 +66,7 @@
     "gulp-sass": "^4.0.2",
     "gulp-sourcemaps": "^2.6.5",
     "gulp-theo": "^2.0.0",
+    "hsluv": "^0.1.0",
```

Comment:
This will have to move to a dependency because it is part of the color factory which will be exported.